clj-http-lite/src/simple/main.clj

```diff
@@ -0,0 +1,19 @@
+(ns simple.main
+  (:require [clj-http.lite.client :as client])
+  (:gen-class))
+
+(def payload "{
+  \"id\": 101,
+  \"title\": \"GraalVM\",
+  \"body\": \"Looks like clj-http.lite works on GraalVM\",
+  \"userId\": 1
+    }")
+
+(defn -main
+  "I don't do a whole lot ... yet."
+  [& args]
+  (println "GET http://example.com/ => Status: " (:status (client/get "http://example.com/")))
+  (println "POST http://example.com/ => Status: " 
+    (:status (client/post "http://example.com/" {:headers {"content-type" "application/json; charset=utf-8"}
```

Comment:
Rather than relying on example.com (which I don't think it exists), I think it would be better to have a docker-compose with a `nginx` with a static file with content that you expect

Reply:
Fair point. I've made the changes. It now uses an nginx image for the post and get requests.

Fun fact example.com is real and reserved by [IANA](https://www.iana.org/domains/reserved)

Reply:
👍